specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -256,18 +256,61 @@
                 }
             }
         },
-        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/dataConnectors": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/alertRules/{ruleId}/actions": {
             "get": {
-                "x-ms-examples": {
-                    "Get all data connectors.": {
-                        "$ref": "./examples/dataConnectors/GetDataConnectors.json"
+                "x-ms-examples": {},
```

Comment:
There are a lot of empty `x-ms-examples`, when will the missing examples be added?

Reply:
The example was in the folder but the references were missing (fix it)